### PR TITLE
Merge ReQUIAM_csv org (hotfix/v0.16.1) into feature/116_grouper25

### PR DESCRIPTION
**Pull request by astrochun**
- Update figshare.ini for repo org change
- Update README.md for org change [ci skip]
- Bump version: v0.16.0 -> v0.16.1

**Description**

See #137

### CODE REVIEW

**Review by astrochun — COMMENTED** [review]
This branch has diverged so additional commits are present. However, this is welcome since this is closer to the production version

File: README.md [hunk 1]
```diff
@@ -296,12 +296,19 @@ Currently, there are two GitHub Action workflows:
 A list of released features and their issue number(s).
 List is sorted from moderate to minor revisions for reach release.
 
-v0.15.0:
+v0.16.0:
+ * Merge `grouper_admin` and `grouper_query` modules #87
+ * Complete adoption of f-strings #118
+ * New pull request templates #120
+ * CI build testing for Python 3.9 #121
```
Comment (damian-romero) [inline README.md]:
```suggestion
v0.16.0 - v0.16.1:
 * Merge `grouper_admin` and `grouper_query` modules #87
 * Complete adoption of f-strings #118
 * New pull request templates #120
 * CI build testing for Python 3.9 #121
 * Update files to reflect ReQUIAM_csv org change #137
```

**Review by damian-romero — COMMENTED** [review]
I added one suggestion but I can merge it myself.

File: README.md (hunk 1 above)
Comment (astrochun) [reply README.md]: Hotfix has this fix. See #138

**Pull request merged**